google-cloud-datastore/src/main/java/com/google/cloud/datastore/Datastore.java

```diff
@@ -505,9 +507,7 @@ interface TransactionCallable<T> {
    * @throws DatastoreException upon failure
    * @return {@link AggregationResults}
    */
-  default AggregationResults runAggregation(AggregationQuery query, ReadOption... options) {
-    throw new UnsupportedOperationException("Not implemented.");
-  }
+  AggregationResults runAggregation(AggregationQuery query, ReadOption... options);
```

Comment:
Hi Kristen, just wondering if the exception is still needed?

Reply:
hi! Since we are removing "default", we cannot add any implementation/exception here. This was used as a workaround previously since we cannot add/update interfaces without adding a 'default' implementation to avoid a breaking change. Since we are now annotating the interface as `@InternalExtensionOnly`, we are able to make such changes and no longer need to have this.

Reply:
> hi! Since we are removing "default", we cannot add any implementation/exception here. This was used as a workaround previously since we cannot add/update interfaces without adding a 'default' implementation to avoid a breaking change. Since we are now annotating the interface as `@InternalExtensionOnly`, we are able to make such changes (per the docs, "An interface marked with this annotation can have new methods added to it.") and no longer need to have this.

Got it! That makes sense! Thanks Kristen!